Sources/Passes/Models/Concrete Models/Pass.swift

```diff
@@ -11,9 +11,11 @@ import FluentKit
 /// The `Model` that stores PassKit passes.
 ///
 /// Uses a UUID so people can't easily guess pass serial numbers.
-open class PKPass: PassModel, @unchecked Sendable {
+final public class Pass: PassModel, @unchecked Sendable {
```

Comment:
Do we want to add a few lines about migrating the models and migrations since the names have changed?

Reply:
The migration has changed name though so Fluent will try and run it again. We can override the migration name though to make it the same

Reply:
But it shouldn't cause big problems since the SQL command that migrations run is `CREATE TABLE IF NOT EXISTS`, right? Besides, this is still a pre-1.0 unstable package, if you are ok with that I would leave it as is